proxy/api/src/http/project.rs

```diff
@@ -49,13 +50,36 @@ fn get_filter(ctx: context::Ctx) -> impl Filter<Extract = impl Reply, Error = Re
         .and_then(handler::get)
 }
 
-/// `GET /`
-fn list_filter(ctx: context::Ctx) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
-    http::with_context(ctx)
+/// `GET /tracked`
+fn tracked_filter(
+    ctx: context::Ctx,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    path("tracked")
+        .and(warp::get())
+        .and(http::with_context(ctx))
+        .and(path::end())
+        .and_then(handler::list_tracked)
+}
+
+/// `GET /contributed`
+fn contributed_filter(
+    ctx: context::Ctx,
+) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
+    path("contributed")
+        .and(warp::get())
+        .and(http::with_context(ctx))
+        .and(path::end())
+        .and_then(handler::list_contributed)
+}
+
+/// `GET /user/<id>
+fn peer_filter(ctx: context::Ctx) -> impl Filter<Extract = impl Reply, Error = Rejection> + Clone {
```

Comment:
So right now we're not differentiating `tracked` vs `contributed` for other users. The creation of this endpoint was due to splitting it out from the "current user" endpoints. 

I'll create a ticket to follow up with your thoughts, but I don't think this work should be done in this PR.

Reply:
> So right now we're not differentiating tracked vs contributed for other users

Do you mean that we only differentiate it for the local user at the moment?

In any case, we are differentiating it for other users since this endpoint now filters specifically tracked users for a given user.

Note that my thought only involves namespace this endpoint with its semantics, just like we do for the local user, so I don't think it should be postponed.

Reply:
> Do you mean that we only differentiate it for the local user at the moment?

Yes, that's the current change in this PR.

> In any case, we are differentiating it for other users since this endpoint now filters specifically tracked users for a given user.

Hmmm so you're talking about the _user_ being tracked and not the tracked user's _projects_?

If so, there's currently no such thing as non-tracked user in our local view. We have no way of knowing about users that are not in our monorepo :)

Reply:
Ah, yes! That's where my confusion steemed. All clear, thanks for bearing with me :)

Reply:
Ok, solved in https://github.com/radicle-dev/radicle-upstream/pull/866#discussion_r490071679 :100: